src/content/blog/astro-5-beta.mdx

```diff
@@ -35,6 +35,12 @@ pnpm install astro@beta
 yarn add astro@beta
 ```
 
+To start a new project using v5-beta, use the `create astro` command for your package manager:
```

Comment:
Elsewhere in the blog post it always says “5.0 beta”, should this be consistent?
```suggestion
To start a new project using the 5.0 beta, use the `create astro` command for your package manager:
```